tests/security/attributes/publisher.cpp

```diff
@@ -238,12 +238,9 @@ int run_test(int argc, ACE_TCHAR *argv[], Args& my_args)
 int ACE_TMAIN(int argc, ACE_TCHAR *argv[])
 {
   Args my_args;
-
-  int result = run_test(argc, argv, my_args);
-  if (result == my_args.expected_result_) {
-    return 0;
-  } else {
+  if (run_test(argc, argv, my_args) != my_args.expected_result_) {
     std::cerr << "Publisher exiting with unexpected result: " << result << std::endl;
```

Comment:
looks like a compile error here... need to add result back in for the error message.